README.md

```diff
@@ -6,6 +6,14 @@
 [![Apache 2 License](https://img.shields.io/packagist/l/aws/aws-sdk-php.svg?style=flat)](http://aws.amazon.com/apache-2-0/)
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/aws/aws-sdk-php?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+
+##Diff
+for some endpoint not supported Signature-v4 yet.
```

Comment:
I don't believe this is accurate for any S3 endpoint. Also, this is not the appropriate place for a changelog diff.